### ISSUE TITLE
Running dask-cudf test showing ModuleNotFoundError: No module named 'cudf.bindings'

### ISSUE THREAD

**Issue by lk1983823** [rapidsai/cudf#5651]
I have successfully installed upgraded dask-cudf following "https://github.com/rapidsai/dask-cudf". However, when I try to test it using py.test dask_cudf, it shows:
```
_______________ ERROR collecting dask_cudf/io/tests/test_csv.py ________________
ImportError while importing test module '/media/lksgcc/new_disk/lk_git/dask-cudf/dask_cudf/io/tests/test_csv.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
dask_cudf/__init__.py:1: in <module>
    from .core import (
dask_cudf/core.py:24: in <module>
    import cudf.bindings.reduce as cpp_reduce
E   ModuleNotFoundError: No module named 'cudf.bindings'
```
Anyone can help? Thanks

**Comment by lk1983823** [comment]
After I build another environment and install it again from source.  The testing error shows: 
```
ERROR collecting dask_cudf/tests/test_sort.py _______________________________________________________________________________
/home/lksgcc/.pyenv/versions/anaconda3-5.0.1/lib/python3.6/site-packages/py/_path/local.py:662: in pyimport
    __import__(modname)
dask_cudf/__init__.py:1: in <module>
    from .core import (
dask_cudf/core.py:8: in <module>
    import dask.dataframe as dd
/home/lksgcc/.pyenv/versions/anaconda3-5.0.1/lib/python3.6/site-packages/dask/dataframe/__init__.py:3: in <module>
    from .core import (DataFrame, Series, Index, _Frame, map_partitions,
/home/lksgcc/.pyenv/versions/anaconda3-5.0.1/lib/python3.6/site-packages/dask/dataframe/core.py:40: in <module>
    pd.core.computation.expressions.set_use_numexpr(False)
E   AttributeError: module 'pandas.core.computation' has no attribute 'expressions'
```
Even when I pip install --upgrade dask to 2.20.0, the error still exists.
My cuda version is 10.1 and pandas 0.25.3

**Comment by galipremsagar** [comment]
Looks like you shouldn't be getting this error on dask 2.20.0. https://github.com/pandas-dev/pandas/issues/18355

Can you try running this and share what version it shows:

```python
import dask
dask.__version__
```

**Comment by galipremsagar** [comment]
From the stack-trace you have posted, the issue is already fixed in dask 2.20.0.

https://github.com/dask/dask/blob/2.20.0/dask/dataframe/core.py#L79

Maybe you are having multiple versions of dask in your `lib` directory - A manual delete of all those versions of dask or a clean create & install of a new environment should fix this issue. I have verified this issue doesn't exist in `dask-2.20.0`(installing both from source and conda).

**Comment by kkraus14** [comment]
Fixed as of dask 2.20.0
